src/screens/WhotoFollow/elements/ItemUser.js

```diff
@@ -39,7 +40,7 @@ const ItemUser = ({photo, username, bio, followed, onPress, userid}) => {
             <IconCheck
               width={dimen.normalizeDimen(32)}
               height={dimen.normalizeDimen(32)}
-              fill="#23C5B6"
+              fill={colors.signed_primary}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on replacing the hardcoded color value with a reference to a centralized color configuration. This will make it easier to maintain and update the color scheme in the future.
```diff
- fill="#23C5B6"
+ fill={colors.signed_primary}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->